core/src/test/java/org/opensearch/sql/planner/PlannerTest.java

```diff
@@ -119,6 +119,16 @@ protected PhysicalPlan analyze(LogicalPlan logicalPlan) {
 
   protected class MockTable extends LogicalPlanNodeVisitor<PhysicalPlan, Object> implements Table {
 
+    @Override
+    public boolean isExist() {
```

Comment:
```suggestion
    public boolean exists() {
```
Is more conventional in Java.

Reply:
yeah, `if table.exists` looks more natural. fixed. thanks!